ddl/error.go

```diff
@@ -273,4 +273,6 @@ var (
 
 	// ErrUnknownEngine is returned when the table engine is unknown.
 	ErrUnknownEngine = dbterror.ClassDDL.NewStd(mysql.ErrUnknownStorageEngine)
+
+	errExchangePartitionDisabled = dbterror.ClassDDL.NewStdErr(mysql.ErrUnsupportedDDLOperation, parser_mysql.Message("Exchange Partition is disabled, please set 'tidb_enable_exchange_partition' if you need to need to enable it", nil))
```

Comment:
Is it indented to use `errExchangePartitionDisabled`? Why not the exported `Err`?

Reply:
Because it will only be used in the current package, just like ` errTablePartitionDisabled`, so I don’t think it needs to be exported.